test/ziggurat/metrics_test.clj

```diff
@@ -1,8 +1,29 @@
 (ns ziggurat.metrics-test
   (:require [clojure.test :refer :all]
+            [ziggurat.config :refer [ziggurat-config]]
+            [ziggurat.fixtures :as fix]
             [ziggurat.metrics :as metrics])
   (:import (io.dropwizard.metrics5 Meter Histogram UniformReservoir)))
 
+(use-fixtures :once fix/mount-only-config-and-producer)
```

Comment:
why is the producer being started here?

Reply:
nice catch, I will mount only the config